reflectwalk.go

```diff
@@ -4,9 +4,7 @@
 // those elements.
 package reflectwalk
 
-import (
-	"reflect"
-)
+import "reflect"
```

Comment:
As I stylistic thing, I usually just use the `import ()` syntax cause its way less annoying when I need to add/remove packages (often things like fmt for testing). But I don't care really.